packages/zoe/src/contractSupport/ratio.js

```diff
@@ -162,6 +194,7 @@ export const addRatios = (left, right) => {
   );
 };
 
+/** @type {MultiplyRatios} */
```

Comment:
AIUI, these need the declarations because the functions are being exported independently. If they were methods, or even functions within a record, we could declare the containing object, and typescript would figure out the individual members. I don't know how to make that happen with independent exports.

Reply:
I agree with Chris here. It's part of our current standard practice to add a type declaration to functions which references a type defined in a `types.js` file.